nanoc/lib/nanoc/filters/sass/importer.rb

```diff
@@ -20,15 +20,16 @@ def find_relative(name, base_identifier, options)
       return unless raw_filename
 
       item = raw_filename_to_item(raw_filename)
-      # it doesn't make sense to import a file, from Nanoc's content if the corresponding item has been deleted
-      raise "unable to map #{raw_filename} to any item" if item.nil?
```

Comment:
Alright, this line indeed makes the compilation stop.

I think you should replace
```
# it doesn't make sense to import a file, from Nanoc's content if the corresponding item has been deleted
raise "unable to map #{raw_filename} to any item" if item.nil?
```
by
```
# the imported file doesn't correspond to a Nanoc item, let next filter in Sass' :load_paths handle it
return if item.nil?
```

And keep the rest of `importer.rb` as it is.

This suggestion alone fixes #1378.

Reply:
As discussed on Gitter, this won’t work with paths that are relative to the item itself and live outside of the Nanoc content directory.

e.g. current item = `content/style/foo.sass`, path to import = `../../external.sass'